app/services/reports/region_service.rb

```diff
@@ -10,34 +10,48 @@ def self.call(*args)
     end
 
     def initialize(region:, period:, months: MAX_MONTHS_OF_DATA)
-      @current_user = current_user
       @region = region
+      @slug = @region.slug
       @period = period
       start_period = period.advance(months: -(months - 1))
       @range = Range.new(start_period, @period)
     end
 
-    attr_reader :current_user
     attr_reader :result
     attr_reader :period
     attr_reader :range
     attr_reader :region
+    attr_reader :slug
 
     def call
-      result = ControlRateService.new(region, periods: range).call
+      result = Reports::Result.new(region: @region, period_type: @range.begin.type)
+      result.adjusted_patient_counts = repository.adjusted_patients_without_ltfu[slug]
+      result.adjusted_patient_counts_with_ltfu = repository.adjusted_patients_with_ltfu[slug]
+      result.assigned_patients = repository.assigned_patients[slug]
+      result.controlled_patients = repository.controlled[slug]
+      result.controlled_patients_rate = repository.controlled_rates[slug]
+      result.controlled_patients_with_ltfu_rate = repository.controlled_rates(with_ltfu: true)[slug]
+      result.cumulative_assigned_patients = repository.cumulative_assigned_patients[slug]
+      result.cumulative_registrations = repository.cumulative_registrations[slug]
+      result.earliest_registration_period = repository.earliest_patient_recorded_at_period[slug]
+      result.ltfu_patients = repository.ltfu[slug]
+      result.ltfu_patients_rate = repository.ltfu_rates[slug]
+      result.registrations = repository.monthly_registrations[slug]
+      result.uncontrolled_patients = repository.uncontrolled[slug]
+      result.uncontrolled_patients_rate = repository.uncontrolled_rates[slug]
+      result.uncontrolled_patients_with_ltfu_rate = repository.uncontrolled_rates(with_ltfu: true)[slug]
```

Comment:
Since we've been replacing things w/ calls to Repository, this was basically all that `ControlRateService` was doing...we were just passing hashes back and forth.  So we directly populate the Result object here now with the values from the repo.